### PR TITLE
added attribute `terminationGracePeriodSeconds`

### PR DESCRIPTION
**Pull request by hcsu**
Add attribute `terminationGracePeriodSeconds`

### CODE REVIEW

File: jsonnetlib/k8s.jsonnet [hunk 1]
```diff
@@ -314,6 +314,7 @@ cfg {
     env=[],
     container={},
     volumes=[],
+    terminationGracePeriodSeconds=30,
```
Comment (idealhack) [inline jsonnetlib/k8s.jsonnet:317]:
```suggestion
    terminationGracePeriodSeconds=root.terminationGracePeriodSeconds,
```

File: jsonnetlib/k8s.jsonnet [hunk 2]
```diff
@@ -451,6 +453,7 @@ cfg {
     maxSurge=root.maxSurge,
     container={},
     volumes=[],
+    terminationGracePeriodSeconds='',
```
Comment (idealhack) [inline jsonnetlib/k8s.jsonnet:456]:
```suggestion
    terminationGracePeriodSeconds=root.terminationGracePeriodSeconds,
```

**Review by idealhack — CHANGES_REQUESTED** [review]

File: jsonnetlib/k8s.jsonnet (hunk 1 above)
Comment (idealhack) [reply jsonnetlib/k8s.jsonnet:317]: please move the default value to https://github.com/theplant/plantbuild/blob/master/jsonnetlib/config.jsonnet

Comment (hcsu) [reply jsonnetlib/k8s.jsonnet]: Thanks, moved to config.jsonnet.

File: jsonnetlib/k8s.jsonnet (hunk 2 above)
Comment (hcsu) [reply jsonnetlib/k8s.jsonnet]: Thanks, moved to config.jsonnet.

**Comment by hcsu** [comment]
Successful deployed `terminationGracePeriodSeconds=2333` on test-group-c/ord-foe-order

CleanShot 2021-07-06 at 18 22 41@2x

**Pull request merged**